setup/setupEnvironment.sh

```diff
@@ -249,10 +279,11 @@ grep "=" ../src/main/kotlin/com/budilov/Properties.kt
 echo "-Try out the following commands: -"
 
 echo "Upload a picture"
-echo "aws s3 cp new-york.jpg s3://${BUCKET_NAME}/usercontent/us-east-1:11122233-4455-6677-8888-999999999999/"
-
+echo "aws s3 cp new-york.jpg s3://${BUCKET_NAME}/usercontent/${COGNITO_POOL_ID}/"
```

Comment:
ok -- I need to review the cognito code -- I've got a misunderstanding of whats going on somewhere . . . 

Thanks for the help!

tom